Jenkinsfile

```diff
@@ -10,13 +10,13 @@ elifePipeline {
         {
             stage 'Build images', {
                 checkout scm
-                dockerComposeBuild(commit)
+                dockerComposeBuild commit
             }
 
             stage 'Project tests', {
                 def coverallsToken = sh(script:'cat /etc/coveralls/tokens/profiles', returnStdout: true).trim()
                 withEnv(["COVERALLS_REPO_TOKEN=$coverallsToken"]) {
-                    dockerComposeProjectTests('profiles', commit, ['/srv/profiles/build/*.xml'])
+                    dockerComposeProjectTests 'profiles', commit
```

Comment:
seems to have lost artifacts as https://alfred.elifesciences.org/blue/organizations/jenkins/pull-requests-projects%2Fprofiles/detail/PR-88/14/tests is empty. Was this failing?

Reply:
Looks like an oversight. Also looks like this file doesn't actually need to change? Reverting.